### DISCUSSION TITLE
keeping sync to the domain via RFC 2136?

### DISCUSSION THREAD

**Discussion by ser** [happyDomain/happydomain#31] [General]
I noticed that if I add a host record directly using knotc, the content of this domain is not updated to happydomain, so happydomain wants to delete the record added.

is there any way for happydomain to keep in sync with content of the domain records modified by external tools?

**Comment by nemunaire** [comment]
Hi @ser,

Yes, you can use the action "Fetch current deployed zone" in the menu:
<img src="https://github.com/user-attachments/assets/1c208077-f800-4328-a0cc-bc6193e9f0a7" width="200" />

This will retrieve the zone as it is currently on the main domain name server. This will also erase any modifications made in happyDomain and not published, there is no kind of merge.

Were you looking for something like this? Or did you have another behavior in mind?

**Comment by ser** [comment]
Thank you, I have not noticed that option in the menu. It works as it should, indeed.
